ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/DumpMetaData.java

```diff
@@ -203,7 +204,7 @@ private void initializeIfNot() throws SemanticException {
     }
   }
 
-  private List<String> prepareReplScopeValues() {
+  public List<String> prepareReplScopeValues() {
```

Comment:
why is this needed?

Reply:
Done to give access to this function from the unit-test (testDumpMetadataBackwardCompatibility). This function is required to create the repl-scope values during dump. So, during the test, while re-creating the dmd file in the same manner, this function is used.